backend/action_logs/serializers.py

```
@@ -123,4 +133,43 @@ def to_representation(self, instance):
+        model = ActionLogNotification
+        fields = ['id', 'user', 'action_log', 'comment', 'is_read', 'created_at']
+        read_only_fields = ['id', 'user', 'action_log', 'comment', 'is_read', 'created_at']
+
+class ActionLogCommentSerializer(serializers.ModelSerializer):
+    user = UserSerializer(read_only=True)
+    replies = serializers.SerializerMethodField()
+    parent_id = serializers.IntegerField(write_only=True, required=False)
+
+    class Meta:
+        model = ActionLogComment
+        fields = ['id', 'action_log', 'user', 'comment', 'created_at', 'updated_at', 'parent_id', 'replies']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)